packages/ra-core/src/form/FormGroupContext.ts

```diff
@@ -0,0 +1,5 @@
+import { createContext } from 'react';
+
+export const FormGroupContext = createContext<FormGroupContextValue>(undefined);
```

Comment:
you should add a jsdoc saying that this shouldn't be used in standalone, but through FormGroupContextProvider